cleanlab/count.py

```diff
@@ -510,6 +470,104 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
+    input `labels` is a list of lists (or list of iterable).
+    This is intended as a helper function. You should probably
+    be using `compute_confident_joint(multi_label=True)` instead.
+
+    The MAJOR DIFFERENCE in how this is computed versus single_label,
+    is the total number of errors considered is based on the number
+    of labels, not the number of examples. So, the confident_joint
+    will have larger values.
+
+    See `compute_confident_joint` docstring for more info.
+
+    Parameters
+    ----------
+    labels : list of list/iterable (length N)
+        These are multiclass labels. Each list in the list contains
+        all the labels for that example. This method will fail if labels
+        is not a list of lists (or a list of np.ndarrays or iterable).
+
+    pred_probs : np.ndarray (shape (N, K))
+        P(label=k|x) is a matrix with K model-predicted probabilities.
+        Each row of this matrix corresponds to an example `x` and contains the model-predicted
+        probabilities that `x` belongs to each possible class.
+        The columns must be ordered such that these probabilities correspond to class 0, 1, 2,..., K-1.
+        `pred_probs` must be out-of-sample (ideally should have been computed using 3+ fold cross-validation).
+
+    thresholds : iterable (list or np.ndarray) of shape (K, 1)  or (K,)
+        P(label^=k|label=k). If an example has a predicted probability "greater" than
+        this threshold, it is counted as having true_label = k. This is
+        not used for filtering/pruning, only for estimating the noise rates using
+        confident counts. This value should be between 0 and 1. Default is None.
+
+    calibrate : bool, default = True
+        Calibrates confident joint estimate P(label=i, true_label=j) such that
+        np.sum(cj) == len(labels) and np.sum(cj, axis = 1) == np.bincount(labels).
+
+    return_indices_of_off_diagonals: bool, default = False
+        If true returns indices of examples that were counted in off-diagonals
+        of confident joint as a baseline proxy for the label issues. This
+        sometimes works as well as filter.find_label_issues(confident_joint).
+
+    Returns
+    -------
+    confident_joint_counts : np.ndarray
+      An array of shape ``(num_classes,2, 2)`` representing the confident joint, the matrix used for identifying label issues, which
+      estimates a confident subset of the joint distribution of the noisy and true labels, ``P_{noisy label, true label}``.
+      Entry ``(c, j, k)`` in the matrix is the number of examples in a one-vs-rest class confidently counted into the pair of ``(class c, noisy label=j, true label=k)`` classes.
+
+    Note: if return_indices_of_off_diagonals is True, this function returns confident_joint_counts, indices_off_diagonal
+    where indices_off_diagonal is a list of array of indices counted in off-diagonals of confident joint for each class.
```

Comment:
```suggestion
    Note: if `return_indices_of_off_diagonals` is set as True, this function instead returns a tuple `(confident_joint_counts, indices_off_diagonal)`
    where `indices_off_diagonal` is a list of arrays (one per class) and each array contains the indices of examples counted in off-diagonals of confident joint for that class.
```

Reply:
note how we should always use special formatting when referring to variables in docstrings. And for referring to general code blocks, should use double-backticks. Our packages uses the numpy documentation style format, so please look through the relevant links in Development.md to learn how that works.

Reply:
can you also copy this improved part of the docstring (the part that is not specific to multilabel) to the docstring return-type description above for `compute_confident_joint()` ?